osquery/tables/system/posix/authorized_keys.cpp

```diff
@@ -48,13 +68,78 @@ void genSSHkeysForUser(const std::string& uid,
       logger.log(google::GLOG_WARNING, s.getMessage());
       continue;
     }
+
     // Protocol 1 public key consist of: options, bits, exponent, modulus,
     // comment; Protocol 2 public key consist of: options, keytype,
     // base64-encoded key, comment.
     for (const auto& line : split(keys_content, "\n")) {
       if (!line.empty() && line[0] != '#') {
-        Row r = {{"uid", uid}, {"key", line}, {"key_file", keys_file.string()}};
-        r["pid_with_namespace"] = "0";
+        auto splitted_line = split(line, " ");
+        if (splitted_line.empty()) {
+          // Line syntax is invalid.
+          continue;
+        }
```

Comment:
there's a check for `line.empty()` above, is this likely to happen?

Reply:
I've deleted this check, after a second thought, it's redundant.